README.md

```diff
@@ -242,6 +228,32 @@ The transaction object can give you the tx_hash of the transaction before sendin
 tx.release()
 ```
 
+### Whitelist a transaction
+```python
+# Assuming you are registered as a whitelisted digital service with the Kin Ecosystem (exact details TBD)
+# You will be able to whitelist transactions for your clients, making it so that their fee will not be deducted
+# Your clients will send an http request to you containing their tx.
+# You can then whitelist it, and return it back to the client to send to the blockchain
```

Comment:
maybe add more detail what's the whitelist does (adding signature) so the developer will understand that a specific account/accounts are whitelisted and those the ones that should be used for whitelisting.
As a side note, what will happen with channels and whitelisting? does any channel account considered whitelisted as well? (signed by the parent account or something?)

Reply:
Channel transactions are signed by the parent so they are whitelisted.
These docs will be edited to be more professional and will include all explinations